src/dom/index.js

```diff
@@ -31,7 +31,7 @@ export function removeNode(node) {
  *	@private
  */
 export function setAccessor(node, name, value, old, isSvg) {
-	ensureNodeData(node)[name] = value;
+	node[ATTR_KEY][name] = value;
```

Comment:
Only `diffAttributes` calls `setAccessor` (in 2 different cases) so I moved the `ensureNodeData` call into `diffAttributes` so that it only gets called once.

`idiff` always calls `diffAttributes`, which means `ensureNodeData` will always be called.